forc-plugins/forc-client/src/cmd/deploy.rs

```diff
@@ -57,9 +57,6 @@ pub struct Command {
     pub unsigned: bool,
     /// Set the key to be used for signing.
     pub signing_key: Option<SecretKey>,
-    /// Sign the deployment transaction manually.
```

Comment:
Why is this removed?

Reply:
That is a good catch! I will remove this in a follow-up PR as this is not a meaningful anymore but it is a breaking change